### PR TITLE
Add listing endpoints for thumbnails

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request
Related to 
https://github.com/pimcore/studio-ui-bundle/issues/80
https://github.com/pimcore/studio-ui-bundle/issues/214

## Additional info
Added endpoints to list:
- all video thumbnails
- all downloadable image thumbnails

these are used in the sidebars of images and videos

### CODE REVIEW

File: src/Thumbnail/Schema/Thumbnail.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+/**
+ * @internal
+ */
+#[Schema(
+    title: 'Thumbnail',
+    required: ['id', 'text'],
+    type: 'object'
+)]
+final readonly class Thumbnail
```
Comment (mattamon) [inline src/Thumbnail/Schema/Thumbnail.php]: Please add the additional attributes here and the thumbnail event :)

Comment (lukmzig) [reply src/Thumbnail/Schema/Thumbnail.php]: I was just waiting till the Auth will be merged and forgot to convert this PR to draft, but I will do the changes now :)

File: src/OpenApi/Config/Tags.php [hunk 2]
```diff
@@ -25,9 +25,13 @@
...
     description: 'Asset operations to get/update/create/delete assets'
 )]
+#[Tag(
+    name: Tags::Thumbnails->name,
+    description: 'List thumbnails for assets like videos and images'
+)]
 #[Tag(
     name: Tags::Authorization->name,
-    description: 'Login via username and password to get a token or refresh the token'
+    description: 'Login via username and password'
```
Comment (mattamon) [inline src/OpenApi/Config/Tags.php]: Should we maybe state that this is now sessions based and you can also logout?

Comment (lukmzig) [reply src/OpenApi/Config/Tags.php]: Updated now :)

**Pull request merged**